### ISSUE TITLE
Is the code data used in the training data?

### ISSUE THREAD

**Issue by xuhu0115** [simplescaling/s1#112]
Hello, may I ask if code data was used in the training data? I noticed that you mentioned using the LiveCodeBench and USACO code datasets, but these two labels do not appear in the 59k source_type. How can this be explained?

**Comment by Muennighoff** [comment]
don't think there code data; the two may have been filtered out due to not
having labels

**Comment by xuhu0115** [comment]
"Thank you for your reply! Additionally, I have another question. The 59k dataset mentions the GPQA dataset as one of its sources. I noticed in the paper that the GPQA Diamond subset was used for evaluation. Could you clarify whether the GPQA subset was also used during training?"

**Comment by Muennighoff** [comment]
it's not the diamond subset that is in the training but its other parts (and also additionally decontaminated against the diamond)

**Comment by xuhu0115** [comment]
Okay, thank you for your answer!
